doc/examples/plot_circular_elliptical_hough_transform.py

```diff
@@ -128,10 +128,10 @@
 
 # Estimated parameters for the ellipse
 best = result[-1]
```

Comment:
I also tried to find another way to write that when I first saw this PR. My hesitation about a combination of map and lambda function is the good knowledge of python it requires. The underlying question is: who is going to read the documentation? not all readers are familiar with pure pythonic styles. Hard to find the right balance I think.